Code-the-Dream-School/ee-prac-team1-front
# PRAC-29 React Dashboard - Activity card

An activity card component has been created.

Testing Steps:
Run the application.
Verify that the activity card component appears.
![image](https://github.com/Code-the-Dream-School/ee-prac-team1-front/assets/109319234/2558ea6d-8a15-441d-8a1b-cb4579881012)
